packages/core/stdlib/math/main.browse

```diff
@@ -23,6 +23,7 @@ rule toPrecision {
 
 # Math.* fns
 rule abs { bind x; return (native:fn abs $x) }
+#* @help { The arc-cos of a number }
```

Comment:
should we use `##` instead of `#*`?

Reply:
I have no strong preference either way

Reply:
That definitely looks better but I chose this because I was thinking of how people like to do

###################
#         Section title      #
###################

Which would trigger the ## check

Reply:
While that's true, it's fine because we have a second layer of `@help...` (i.e. they need to use @ for a tag)

Even in JS, people do this all the time:
```js
/*********************\
 *** Section Title ***
\*********************/
```